Makefile

```diff
@@ -213,7 +213,9 @@ check-translate: locale/circuitpython.pot $(wildcard locale/*.po)
 	$(PYTHON) tools/check_translations.py $^
 
 stubs:
-	rst2pyi $(VALIDATE) shared-bindings/ $(STUBDIR)
+	mkdir -p circuitpython-stubs
+	python tools/extract_pyi.py shared-bindings/ $(STUBDIR)
+	python tools/extract_pyi.py ports/atmel-samd/bindings $(STUBDIR)
 	python setup.py sdist
```

Comment:
these should probably be $(PYTHON), at least the added lines.  Otherwise, systems where the bare `python` program is python2 fail.  This was the case on my usual development system, made out of debian stable.